content/docs/observability/release/_index.md

```diff
@@ -9,15 +9,14 @@ Description: >
 ## Release Notes - CSM Observability 1.6.0
 
 
+
 ### New Features/Changes
 
 - [#724 - [FEATURE]: CSM support for Openshift 4.13](https://github.com/dell/csm/issues/724)
 - [#922 - [FEATURE]: Use ubi9 micro as base image](https://github.com/dell/csm/issues/922)
-- [#926 - [FEATURE]: Set up golangci-lint for all CSM repositories](https://github.com/dell/csm/issues/926)
```

Comment:
K8s 1.28 is missing and duplicate of UBI 9 Micro

Reply:
@rajkumar-palani I can't see https://github.com/dell/csm/issues/947 observability labels on this github issue, are we supporting this feature for observability?